### PR TITLE
Product page #35

### PR DESCRIPTION
**Pull request by nrod80**
(#35) An un-styled product page accessible at '/api/product/:productId'. Allow user to select upgrade options, but no ability to submit those options yet.

### CODE REVIEW

File: browser/js/fsa/fsa-pre-built.js [hunk 1]
```diff
@@ -117,7 +117,7 @@
 
         this.user = null;
 
-        this.create = function (sessionId, user) {
+        this.create = function (user) {
```
Comment (clocasto) [inline browser/js/fsa/fsa-pre-built.js:120]: I'm wondering if this will conflict with Yi's PR at all. We'll need to change this back I think...

**Comment by curiousYi** [comment]
looks good.

File: gulpfile.js [hunk 2]
```diff
@@ -46,9 +46,9 @@ gulp.task('buildJS', ['lintJS'], function () {
         .pipe(plumber())
         .pipe(sourcemaps.init())
         .pipe(concat('main.js'))
-        .pipe(babel({
-            presets: ['es2015']
-        }))
+        // .pipe(babel({
```
Comment (clocasto) [inline gulpfile.js]: I think we are using ES6 features, is this an issue?

File: server/start.js [hunk 3]
```diff
@@ -12,5 +12,5 @@
 
 */
 
-require('babel-register');
+// require('babel-register');
```
Comment (clocasto) [inline server/start.js]: Same as above - I think we may need this?

**Review by clocasto — CHANGES_REQUESTED** [review]
Will the omission of Babel impact other parts of app?

File: browser/js/fsa/fsa-pre-built.js (hunk 1 above)
Comment (clocasto) [reply browser/js/fsa/fsa-pre-built.js:120]: Nevermind - this is a canonical fix.

**Review by clocasto — APPROVED** [review]

**Pull request merged**